packaging/fedora/snapd.spec

```diff
@@ -580,6 +606,9 @@ rm -rfv %{buildroot}%{_sysconfdir}/apparmor.d
 # ubuntu-core-launcher is dead
 rm -fv %{buildroot}%{_bindir}/ubuntu-core-launcher
 popd
+%if %{with apparmor}
+install -m 644 -D cmd/snap-confine/snap-confine.apparmor %{buildroot}%{_sysconfdir}/apparmor.d/usr.libexec.snapd.snap-confine
```

Comment:
Does AppArmor lack a place for system installed profiles separate from user-created ones?

Reply:
With apparmor, there isn't currently a difference between system policy and user policy, but there is a difference between apparmor-managed policy and other application-managed policy. This snap-confine profile is currently considered a system profile, so this is the correct place. The snapd-managed profiles live in /var/lib/snapd/apparmor/profiles. snapd may end up moving snap-confine policy somewhere else, but this is currently where it expects it.